src/classes/Feature.cls

```diff
@@ -40,5 +40,32 @@ public without sharing class Feature {
 		private set;
 	}
 
+	/*
+	* Returns a pod instance for your org
+	* Example: returns na2 if your instance points to https://na2.salesforce.com
+	*/
+	public static String instancePodName {
```

Comment:
1. hmm, needs to be tested. I am not sure what getSalesforceBaseUrl() returns in this instance. Never tried it. 
2. Wish I knew this before, I'd rewrite to use this (sheepish smile) :-)
3. Yes, missed that pattern since I wrote this independently. will rewrite